index.js

```diff
@@ -120,7 +120,7 @@ function terraform(params) {
   }
 
   core.startGroup('Run terraform plan');
-  const tfp = terraform(`plan -out=terraform.plan -lock=${terraformLock} -parallelism=${terraformParallelism}`);
+  const tfp = terraform(`plan -lock=${terraformLock} -parallelism=${terraformParallelism} -out=terraform.plan`);
```

Comment:
Not strictly needed, but I prefer keeping the order the same as with `terraform apply`.